modules/serverless-eventing-components.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * serverless-knative-eventing.adoc
+
+[id="serverless-eventing-components_{context}"]
+= Knative Eventing components
+
+A typical Knative Eventing deployment on {ServerlessProductName} follows this basic workflow.
+
+. Event producers send information about changes in the state of the system to an event source.
+. The event source translates this information into an event, which is then passed to the broker.
+. Once an event has been sent to the broker, triggers can filter which events are routed to a subscription.
+
+Using the broker and trigger Kubernetes Custom Resource Definitions (CRDs) together enables an event delivery mechanism that hides the details of event routing from the event producer and event consumer.
+Events can also be passed directly from an event channel to a broker, or to a subscription directly.
+
+Event source:: Event sources are Kubernetes custom resources which provide a mechanism for registering interest in a class of events from an event producer.
++
+{ServerlessProductName} can be used with Kafka and Camel K event sources, which can be installed using the Operator Hub in the {product-title} web console.
+
+Broker:: A broker represents an event mesh. Events are sent to the broker's ingress and are then sent to any event consumers that are subscribed to that broker.
+Once inside the broker, metadata for the event is stripped away, so that there is no information about how the event entered the broker.
++
+Example YAML file defining a broker:
+----
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: default <1>
+spec:
+  channelTemplateSpec: <2>
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: InMemoryChannel
+----
+
+<1> The `default` broker is provided with your {ServerlessProductName} Knative Eventing installation.
+<2> Event channels can be specified in the `channelTemplateSpec`.
```

Comment:
Do you have any more details about how to set the channel?

Reply:
This will come later. The whole section on Eventing will be commented out atm until it's tech preview, then I'll remove the other github link and publish these modules. There's a lot of content still to add but I'm doing the work in sections.